### DISCUSSION TITLE
LilyGo T-Display ESP32-S3 - Red is orange

### DISCUSSION THREAD

**Discussion by doh38** [Bodmer/TFT_eSPI#2455] [Compatible displays and setup files]
Servus,
I have a LilyGo T-Display ESP32-S3  and I think it uses your Library/drivers-settings. Unfortunately the colors are quite off which is a bit sad seeing the good qualiy of the display.
-To make it short. it's impossible to get an acceptable "red". 0xFB00 is Orange at the best. 
-Light_Grey 0xCDF9 for instance  is quite ok when looking close to the display, but will emit green tint when "looker" goes a bit away from screen. 
I have seen some kind of gamma settings in the ST7789V driver. But I have no Idea how to correct this. Is there some "tutorial/info?" should I put an issue on the T-DISPLAY S3 Github? :)
Or it is an Hardware thing I may have to live with it?
Gruß,

**Comment by Bodmer** [comment]
0xF800 is red.
16 bit colour format is 5 bits red,  bits green and 5 bits blue.

See also the option here in the dedicated setup for that display:
https://github.com/Bodmer/TFT_eSPI/blob/master/User_Setups/Setup206_LilyGo_T_Display_S3.h#L6

If you already have that line in your setup, try with it commented out.

**Reply by doh38** [reply]
Thanks. 
0xFB00 or 0xF800 show the same "red-orange"
commenting the //init_3 makes everything very Blue. 
As mentioned before, the colors are actually pretty ok. only a little bit too greenish and or red are orange :)
Is there a way to "fine tune" the colors? or the gamma? I think there was something with gamma 4-5 settings from 1.8 ->2.5?

**Reply by Bodmer** [reply]
The init code was provided by LilyGo, maybe post a question here:
https://github.com/Xinyuan-LilyGO/T-Display-S3

**Reply by doh38** [reply]
thanks, I posted an issue there :)
